plugin/normalize.go

```diff
@@ -115,11 +115,15 @@ func SplitHostPort(s string) (host, port string, ipnet *net.IPNet, err error) {
 	if err == nil {
 		if rev, e := dns.ReverseAddr(ip.String()); e == nil {
 			ones, bits = n.Mask.Size()
+			sizeDigit := 8
```

Comment:
Can you put a comment here that mirror your commit comment?

Reply:
Sure.  I am adding this comment